go.mod

```diff
@@ -240,4 +240,6 @@ replace (
 	// controller runtime
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20211014063134-be2a7fb8aa44
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
+	// TODO(fail-forward): Remove when api changes are released
+	github.com/operator-framework/api => github.com/awgreene/api v0.0.6-0.20220412182759-c4956d3f11bd
```

Comment:
Why don't we merge the api change first, then point this at the pseudo-version for `master`? I'd rather not depend on a personal fork if we can help it.